frontend/src/old-pages/Configure/Components.tsx

```diff
@@ -659,20 +665,54 @@ function ActionsEditor({basePath, errorsPath}: any) {
   const onConfiguredErrors = useState([...errorsPath, 'onConfigured'])
 
   return (
-    <>
-      <SpaceBetween direction="vertical" size="xs">
-        <ActionEditor
-          label="On Start"
-          errorPath={onStartErrors}
-          path={onStartPath}
-        />
+    <SpaceBetween direction="vertical" size="xs">
+      <ActionEditor
+        label="On Start"
+        errorPath={onStartErrors}
+        path={onStartPath}
+      />
+      <ActionEditor
+        label="On Configured"
+        errorPath={onConfiguredErrors}
+        path={onConfiguredPath}
+      />
+    </SpaceBetween>
+  )
+}
+
+function HeadNodeActionsEditor({basePath, errorsPath}: ActionsEditorProps) {
+  const actionsPath = [...basePath, 'CustomActions']
+  const onStartPath = [...actionsPath, 'OnNodeStart']
+  const onConfiguredPath = [...actionsPath, 'OnNodeConfigured']
+  const onUpdatedPath = [...actionsPath, 'OnNodeUpdated']
+
+  const onStartErrors = useState([...errorsPath, 'onStart'])
+  const onConfiguredErrors = useState([...errorsPath, 'onConfigured'])
+  const onUpdatedErrors = useState([...errorsPath, 'onUpdated'])
+
+  const version = useState(['app', 'version', 'full'])
+  const isOnNodeUpdatedActive = isFeatureEnabled(version, 'on_node_updated')
+
+  return (
+    <SpaceBetween direction="vertical" size="xs">
+      <ActionEditor
+        label="On Start"
+        errorPath={onStartErrors}
+        path={onStartPath}
+      />
+      <ActionEditor
+        label="On Configured"
+        errorPath={onConfiguredErrors}
+        path={onConfiguredPath}
+      />
```

Comment:
Question: isn't there a way to avoid repetition of code following DRY (between HeadNodeActionsEditor and ActionsEditor) ?

Reply:
We can avoid code duplication by leveraging a flag in `ActionsEditor` component to show/hide the "On Updated" editor, but honestly I am more inclined to the solution I proposed.